DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateApprovals/Index.cshtml

```diff
@@ -27,9 +28,12 @@
       }
       <div class="nhsuk-grid-row nhsuk-u-margin-top-7">
         <div class="nhsuk-grid-column-full">
-          <a class="nhsuk-button" href="" role="button">
-            Approve all
-          </a>
+          <form asp-controller="DelegateApprovals" asp-action="ApproveDelegatesForCentre" method="post">

```

Comment:
Is there a more sensible way to get these post requests out without any JS? This works smoothly but feels like a hack.

Reply:
Do you mean in terms of having an individual form per button? The form with an asp-action is the standard way to do it.

The only other sensible way to do multiple posts per page with no JS is with the stuff I have been doing for the My Account Edit Details and Registration Prompt answer editing, but the workaround found for those was only needed due to the requirement to have forms within forms. I can give you a quick run through of that approach and why it was needed if you want, but I think the current approach here with multiple form elements is better.